### DISCUSSION TITLE
What is your favorite animal?

### DISCUSSION THREAD

**Discussion by Miyai75** [NAIST-SE/AttractingContributors#1059] [2024 Attraction Contributions]
https://github.com/Miyai75/What-is-your-favorite-animal

**Comment by 37integer** [comment]
cat

**Comment by natsu9731** [comment]
dog

**Comment by gill876** [comment]
Mine is Orca!

Please make a contribution to my project!
https://github.com/NAIST-SE/AttractingContributors/discussions/1089#discussion-6846489

**Comment by KazumaYamasaki** [comment]
I added `Cat`!
I would be happy if you could also send a pull request to my repository!
↓
https://github.com/KazumaYamasaki/AttractingContributors-Keyboard
